files/aws/lib/systemd/system/systemd-networkd.service.d/20-override.conf

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
it looks like this file is present twice.. `20-override.conf` and `aws-override.conf`.. is this intentional?

Reply:
The 20-override.conf has removed. Sorry for the confusion.